.github/workflows/polkadot.yml

```diff
@@ -56,7 +56,7 @@ jobs:
           tag: ${{ github.event.inputs.srtool_tag }}
           chain: ${{ matrix.chain }}
           workdir: "${{ github.workspace }}/repo"
-          profile: "not-existent"
+          profile: ${{ matrix.chain }}
```

Comment:
wuut ?

Reply:
was just trying to get something to trigger. Have changed it to "production"